content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -184,12 +184,13 @@ ServiceAccount. You can request a specific token duration using the `--duration`
 command line argument to `kubectl create token` (the actual duration of the issued
 token might be shorter, or could even be longer).
 
+{{< feature-state feature_gate_name="ServiceAccountTokenNodeBinding" >}}
+
 When the `ServiceAccountTokenNodeBinding` and `ServiceAccountTokenNodeBindingValidation`
-features are enabled and the `KUBECTL_NODE_BOUND_TOKENS` environment variable is set to `true`,
-it is possible to create a service account token that is directly bound to a `Node`:
+features are enabled, it is possible to create a service account token that is directly bound to a `Node`:
 
 ```shell
-KUBECTL_NODE_BOUND_TOKENS=true kubectl create token build-robot --bound-object-kind Node --bound-object-name node-001 --bound-object-uid 123...456
+kubectl create token build-robot --bound-object-kind Node --bound-object-name node-001 --bound-object-uid 123...456
```

Comment:
```suggestion
# Requires kubectl v1.31 or later
kubectl create token build-robot --bound-object-kind Node --bound-object-name node-001 --bound-object-uid 123...456
```
?

I'm OK to merge this without that detail though; we can iterate.

Reply:
opened follow-up in https://github.com/kubernetes/website/pull/47316